### ISSUE TITLE
[E5 - simlm] how to calculate initial score fields ?

### ISSUE THREAD

**Issue by duongkstn** [microsoft/unilm#1149]
Hi @intfloat , I am curious about your initial msmarco scores. How do you get those scores ?

For example in your file `simlm/data/msmarco_distillation/kd_train.jsonl` , the score of positive `doc_id` is mostly higher than negative ones, so are they `log_softmax` of confidence scores of reranking model ?
Let's say confidence scores of a reranking model are `0.9` (positive), `0.5` and `0.1` (two negatives), so to get scores like yours , I would run:
```
import torch.nn.functional as F
reranking_confidences = torch.tensor([0.9, 0.5, 0.1])
scores = F.log_softmax(reranking_confidences, dim=0)  # tensor([-0.7513, -1.1513, -1.5513])
```
Is above code correct ?

But in your file `simlm/data/msmarco_bm25_official/train.jsonl` , I don't know why score of postive `doc_id` always be `-1.0`, and is mostly lower than negative `doc_id`s ?

Do I misunderstand something ? Please let me know. 
Really thanks.

**Comment by intfloat** [comment]
For `simlm/data/msmarco_distillation/kd_train.jsonl`, the scores are the logits of re-ranker model before softmax. Please check out the following code to see how these scores are used. https://github.com/microsoft/unilm/blob/159df56cdc3f59086b192ec0dc13043b05779ab7/simlm/src/models/biencoder_model.py#L69-L71 

For `simlm/data/msmarco_bm25_official/train.jsonl`, the scores are just there to make sure the data format is consistent, they are not used in training.
